ci/docker/premerge.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM microsoft/dotnet:2.2-sdk as build
+WORKDIR /app
+COPY ./tools ./tools/
+COPY ./ci/docker/build-test-entrypoint.sh ./build-test.sh
```

Comment:
Nit: Rename `build-test-entrypoint.sh` to just `entrypoint.sh` and `./build-test.sh` to `./entrypoint.sh`

Makes it clearer as to what is going on